ApeWorX/ape-etherscan
# feat: add support for Scroll network

### What I did
This PR adds support for the Scroll network using Scrollscan as the explorer.

### How I did it
I followed the same pattern as the other chains in this package.

### How to verify it
Updated test case, but couldn't run it locally yet. Need to see if the CI picks it up...
I have tested it manually to use a verified contract on Scroll and it worked as expected.

### TO-DO
~~The ape-scroll package has a version restriction (<0.7.0) that needs to be loosened. I have created a [PR](https://github.com/theref/ape-scroll/pull/1) on that repo. In the meantime the setup.py points to my fork of that package.~~
EDIT: new ape-scroll was released and the setup.py now points to the published version.

### Checklist
